src/stories/Meter.stories.js

```diff
@@ -0,0 +1,120 @@
+import { html } from 'lit';
+import '../components/chart';
+import argTypes from '../common/config/chartArgTypes';
+
+export default {
+  title: 'Charts/Meter',
+  component: 'kd-chart',
+  decorators: [
+    (story) => html` <div style="max-width: 600px;">${story()}</div> `,
+  ],
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/uwR7B1xbaRXA5spwPvzzFO/Florence-Release?node-id=2051%3A93&mode=dev',
+    },
+  },
+  argTypes: argTypes,
+};
+
+const args = {
+  chartTitle: 'Meter Chart',
+  description: 'Chart description.',
+  labels: ['LOW', 'MED', 'HIGH'],
+  datasets: [
+    {
+      data: [30, 30, 30],
+      borderWidth: 2,
+
+      cutout: '80%',
```

Comment:
This should move to `/config/chartTypes/meter.js` also.

Reply:
But `borderWidth` and `cutout` is modifiable by user so I put here.

Reply:
If those are defaults they need to go in the config file. Technically all options are user modifiable, so no need to use them in a story unless there's a specific variant you want to show .

Reply:
oh can they modify the config file ?

Reply:
done

Reply:
> oh can they modify the config file ?

No. They can override any default option though.

Reply:
okay